src/reducers/pending-breakpoints.js

```diff
@@ -21,33 +18,19 @@ import { prefs } from "../utils/prefs";
 
 import type { PendingBreakpoint } from "../types";
 import type { Action, DonePromiseAction } from "../actions/types";
-import type { Record } from "../utils/makeRecord";
-
-export type PendingBreakpointsMap = I.Map<string, PendingBreakpoint>;
-
-export type PendingBreakpointsState = {
-  pendingBreakpoints: PendingBreakpointsMap
-};
-
-export function initialPendingBreakpointsState(): Record<
-  PendingBreakpointsState
-> {
-  return makeRecord(
-    ({
-      pendingBreakpoints: restorePendingBreakpoints()
-    }: PendingBreakpointsState)
-  )();
+
+export type PendingBreakpointsState = { [string]: PendingBreakpoint };
+
+export function initialPendingBreakpointsState(): PendingBreakpointsState {
+  return restorePendingBreakpoints();
 }
 
 function update(
-  state: Record<PendingBreakpointsState> = initialPendingBreakpointsState(),
+  state: PendingBreakpointsState = initialPendingBreakpointsState(),
   action: Action
 ) {
   switch (action.type) {
     case "ADD_BREAKPOINT": {
-      if (action.breakpoint.hidden) {
-        return state;
```

Comment:
This is not required?

Reply:
It is moved to `addBreakpoint`. Thanks for asking